opm/parser/eclipse/EclipseState/Eclipse3DProperties.cpp

```diff
@@ -854,7 +853,17 @@ namespace Opm {
         }
     }
 
-    // static member function
+    
+    const MessageContainer& Eclipse3DProperties::getMessageContainer() const {
```

Comment:
>I don't think that is a good idea.

I don't think either.

If I understood @joakim-hove correct, the problem comes again: if ``GridProperties`` doesn't have the ``MessageContainer`` member itself, and don't pass a ``MessageContainer`` reference to the methods, how does the message transfer from ``intGridProperties`` and ``doubleGridProperties`` to ``Eclipse3DProperties`` without losing and message?  What do you think? @pgdr